docs/src/network_analysis/network_properties.md

```diff
@@ -0,0 +1,35 @@
+# [Caching of Network Properties in `ReactionSystems`](@id network_analysis_caching_properties)
```

Comment:
I'd probably call this file "network_properties_caching" (since that is what it is about).